### PR TITLE
[simdjson] Update to 0.9.2

### LINKED ISSUE #17542: [simdjson]Update  to 0.9.2
simdjson package out of date.

### PR DESCRIPTION
**Pull request by myd7349**
**Describe the pull request**

- What does your PR fix? N/A.

- Which triplets are supported/not supported? Have you updated the CI baseline?
All. Yes.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes.

### CODE REVIEW

File: ports/simdjson/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "simdjson",
-  "version-string": "0.8.2",
+  "version-string": "0.9.2",
```
Comment (JonLiu1993) [inline ports/simdjson/vcpkg.json:3]: @myd7349 ,Please change "version-string" to "version".

File: versions/s-/simdjson.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "b2514daf6e57b24578347ac5a9c3119c0961f1f5",
+      "version-string": "0.9.2",
```
Comment (JonLiu1993) [inline versions/s-/simdjson.json:5]: Please change "version-string" to "version".

File: ports/simdjson/vcpkg.json (hunk 1 above)
Comment (myd7349) [reply ports/simdjson/vcpkg.json]: @JonLiu1993 Thanks for your kind help.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
Fix #17542

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**